README.md

```diff
@@ -19,6 +19,39 @@ None.
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
 | clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
+| clamav_freshclam_conf | Dictionnary of values to set in freshclam configuration file  | false | No |
+| clamav_clamd_conf | Dictionnary of values to set in clamd configuration file  | false | No |
+| clamav_configuration_backup | Shall we backup configuration files when changed | false | No |
+
+### Example ###
+
+```yaml
+clamav_freshclam_conf:
+  DatabaseMirror: ['db.local.clamav.net', 'database.clamav.net']
+  Bytecode: 'true'
+  PrivateMirror: None
+```
+
+would change :
+
+```shell
```

Comment:
I think this will look better with `properties` syntax highlighting:
```suggestion
```properties
```